net-news/sfeed/sfeed-1.2.ebuild

```
@@ -45,4 +49,6 @@ src_install() {
 		DOCPREFIX="${EPREFIX}/usr/share/doc/${P}" \
```

```suggestion
		DOCPREFIX="${EPREFIX}/usr/share/doc/${PF}" \
```
Correct location for the documentation is `PF` not `P` (PF also includes the revision)